extract/action.yml

```diff
@@ -16,13 +16,13 @@ runs:
     - name: Prepare Timestamp for Layer Cache Busting
       shell: bash
       run: |
-        date --iso=ns | tee ${{ inputs.scratch-dir }}/buildstamp
+        date --iso=ns | tee ${{ inputs.cache-source }}/buildstamp
     - name: Prepare Dancefile to Access Caches
       shell: bash
       run: |
         cat > ${{ inputs.scratch-dir }}/Dancefile.extract <<EOF
-        FROM alpine:3.17
-        COPY ${{ inputs.scratch-dir }}/buildstamp buildstamp
+        FROM busybox:1
+        COPY buildstamp buildstamp
```

Comment:
Instead of this, couldn't this be solved by `docker buildx build --no-cache`? This would avoid having to copy files around.

Reply:
certainly worth a try. given how disjunct layer caching and RUN caching is, that could work and it is certainly neater!

I'll give it a whirl on my internal test project